modules/opencensus/src/main/java/org/apache/ignite/spi/metric/opencensus/OpenCensusMetricExporterSpi.java

```diff
@@ -118,6 +120,26 @@
      */
     private Map<String, Measure> measures = new HashMap<>();
 
+    /** Cached histogram metrics intervals names. */
+    private Map<T2<String, long[]>, String[]> histogramNames = new HashMap<>();
```

Comment:
Seems, a map is not cleared on `MetricRegistry` remove.

Reply:
Done. Thank you.